server/models/Upload.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose");
+
+const uploadSchema = new mongoose.Schema({
+  filePath: {
```

Comment:
Maybe consider adding a new field to the profile model as opposed to creating a new document for profile images

Reply:
added new field to Profile.js